buyer-api/test/queues/dataResponseQueue.mock.js

```diff
@@ -0,0 +1,27 @@
+import td from 'testdouble';
+import sinon from 'sinon';
+import test from 'ava';
+
```

Comment:
We should mock `createQueue`, since test don't cover the queue itself but the processor method.